File: Exercicios/C/BasicBank.c
#include <stdio.h>
int main(){
    
int op = 0;//Variável para o usuário escolher a opção que ele vai escolher
float dep = 0.0, saldo = 0.0, saque = 0.0;//variávies para saber o quanto vai depositar ou sacar e depois mudar o saldo com a quantidade dada
do{
        //inicio do menu principal
    printf("1-) Depositar\n");
    printf("2-) Sacar\n");
    printf("3-) Listar saldo\n");
    printf("4-) Sair do programa\n");
    printf("\nEscolha uma opcao: ");
    scanf("%d",&op);
        //fim do menu principal
    switch(op){
    case 1://se escolher depósito
        printf("\n---------------------------\n");
        printf("Quanto deseja depositar? R$");
        scanf("%f",&dep);
        printf("---------------------------\n");
        if (dep<0){//se o depósito for menor que zero o programa vai apresentar um erro.
                printf("\n-----------------------------------------\n");
                printf("Nao eh possivel depositar valor negativo ! ! !");
                printf("\n-----------------------------------------\n");
        }
        else{//mas senão ele vai pegar apenas adicionar o depósito no saldo
            saldo += dep;
            printf("\n----------------------------------------------\n");
            printf("Deposito de R$%.2f realizado com sucesso ! ! !",dep);
            printf("\n----------------------------------------------\n");
        }
        break;
    case 2://se escolher saque
        printf("\n-----------------------\n");
        printf("Quanto deseja saque? R$");
        scanf("%f",&saque);
        printf("\n-----------------------\n");
        if (saque<0)//se o saque fizer o saldo ficar negativo ele vai apresentar um erro
                printf("\n Nao eh possivel sacar valor negativo.");
        else{//senão ele apenas vai retirar o valor do saldo
            if ((saldo - saque)>=0){
                saldo -= saque;
                printf("\n-------------------------------------------\n");
                printf("Saque de R$%.2f realizado com sucesso ! ! !",saque);
                printf("\n-------------------------------------------\n");
            }
            else{
                printf("\n-----------------");
                printf("\nSaque invalido.");
                printf("\n-----------------\n");
            }
        }
        break;
    case 3://se escolher ver o saldo
        printf("\n-----------------------------\n");
        printf("Voce possui R$%.2f em sua conta\n",saldo);
        printf("-----------------------------\n");
        break;
    case 4:
        printf("\n------------------\n");
        printf("Programa encerrado\n");
        printf("------------------\n");
        break;
    default:
        printf("\n---------------------------------\n");
        printf("Voce escolheu uma opcao invalida!");
        printf("\n---------------------------------\n");
    }

}while(op!=4);
}
//---------------
//~~phelipes2000

File: Exercicios/Python/basicbank.py
def main(saldo=0):
    print("--------------------")
    print("1 - [Deposito]")
    print("2 - [Saque]")
    print("3 - [Ver saldo]")
    print("4 - [SAIR]")
    print("--------------------")
    print(" ")
    o = int(input("Escolha uma opcao: "))
    if o == 1:
        print(" ")
        d = float(input("Digite quando vai depositar: "))
        if d < 0:
            print(" ")
            print("Nao eh possivel fazer um deposito negativo ! ! !")
        else:
            print(" ")
            print("Deposito realizado com sucesso!")
            saldo += d
            main(saldo)
    if o == 2:
        d = float(input("Digite o quanto deseja sacar: "))
        s = saldo - d
        if s < 0:
            print("Nao eh possivel sacar, pois o saldo fica negativo ! ! !")
        else:
            print("Saque feito com sucesso ! ! !")
            saldo -= d
            main(saldo)
    if o == 3:
        print(" ")
        print(f'Seu saldo eh R${saldo:.2f}')
    if o == 4:
        print(" ")
        print("Saindo do programa . . .")
    if o <= 0 or o >=5:
        print("")
        print("Opcao invalida.")


main()
#~~phelipes2000